src/routes/home/HomeScreen.tsx

```diff
@@ -56,6 +60,21 @@ const HomeScreen = ({ navigation }: HomeScreenProps): JSX.Element => {
     })
   }
 
+  const navigateToNextExercise = async (item: Discipline): Promise<void> => {
+    getNextExercise(item)
+      .then(async nextExerciseData => {
+        const [disciplineId, exerciseKey] = nextExerciseData
+        const documents = await loadDocumentsById(disciplineId)
```

Comment:
This probably makes clicks/navigation seem really unresponsive. Maybe we need to move document loading into the exercises again/at least support both. This is really a pain in the a** :cry:

Reply:
Other ideas would of course be welcome...

Reply:
If you want I can implement a proposal

Reply:
We also could request documents when rendering home screen :thinking: 
But feel free to implement a proposal